src/Features/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.cs

```diff
@@ -88,7 +93,7 @@ private void Decrement(AsyncToken token)
             }
         }
 
-        public virtual Task CreateWaitTask()
+        public Task CreateWaitTask()
```

Comment:
Hey, since we're here: you can swap this out by returning Task.CompletedTask in the if (counter == 0) branch since that's a thing we have now.

Reply:
changed.